release-notes/next.yml

```diff
@@ -0,0 +1,70 @@
+- release: "35.20220313.1.0"
+  issues:
+    - id: "1048"
+      title: "networking: consider the effects of BOOTIF kernel argument on nm-initrd-generator"
```

Comment:
Should this be called e.g. `description` instead? `title` seems to imply that it's always the title of the GitHub issue, but how we'll describe fixes may be different from how the issue is titled. (Even if to start we do just copy/paste the title.)